src/services/database/__tests__/message-repository.integration.test.js

```diff
@@ -128,31 +128,31 @@ describe('messageRepository', () => {
       });
       await Message.create({
         conversationId,
-        messageId: attachmentMessageId,
-        type: MessageType.ATTACHMENT,
+        messageId: fragmentMessageId,
+        type: MessageType.FRAGMENT,
         receivedAt: null,
       });
-      await updateAttachmentAndCreateItsParts(attachmentMessageId, conversationId, []);
-      const attachment = await Message.findByPk(attachmentMessageId);
+      await updateFragmentAndCreateItsParts(fragmentMessageId, conversationId, []);
+      const fragment = await Message.findByPk(fragmentMessageId);
 
-      expect(attachment.receivedAt).toEqual(now);
+      expect(fragment.receivedAt).toEqual(now);
     });
 
-    it('should not update receivedAt for a given attachment if database update query throws', async () => {
+    it('should not update receivedAt for a given fragments if database update query throws', async () => {
```

Comment:
'a given fragments' needs a sentence restructure, is it 'given fragments' or 'a given fragment'?